data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -1167,7 +1167,7 @@
 
         [message]
             speaker=unit
-            message= _ "The Treasury! Cool, let’s go loot some booty!"
+            message= _ "The Treasury! Excellent, let’s clear it out!"
```

Comment:
This one reads realllllly formally to me.

Reply:
I didn't think it was _that_ formal. Contrast 'The Treasury, what fortune! Let us raid the enemy's coffers, that we may add its wealth and resources unto our own!'

How about 'Excellent, let's clean out the gold!' Or do you prefer something closer to the original like 'Excellent, let's loot the gold' or even a direct copy substituting only 'Cool' for 'Excellent'? I was just considering @CelticMinstrel's opinion on this particular change.

Reply:
It's not really a big deal, I was just letting you know about it. My brain is off so I'm just going to approve and let you decide if you want to mess around with the wording or not.

Reply:
All right, I'll go with 'let's clean out the gold' as something that's hopefully a little less formal-sounding for you.